src/signals/incident/containers/IncidentContainer/selectors.js

```diff
@@ -1,6 +1,17 @@
 import { createSelector } from 'reselect';
 import { initialState } from './reducer';
 
+export const getClassificationData = (category, subcategory, { _links, name, slug, handling_message }) => ({
```

Comment:
With more than two parameters, it would be better to switch to an options object as a single parameter where the keys would then be `category`, `subcategory` and `subcategoryData` (or something similar). Also, add a test.

Reply:
Sure, good point!